storage/redis/adapter.go

```diff
@@ -0,0 +1,101 @@
+package redis
+
+import (
+	"fmt"
+	"os"
+	"strconv"
+
+	r "gopkg.in/redis.v3"
+)
+
+type adapter struct {
+	bufferSize  int
+	redisClient *r.Client
+}
+
+// NewStorageAdapter returns a pointer to a new instance of a redis-based storage.Adapter.
+func NewStorageAdapter(bufferSize int) (*adapter, error) {
```

Comment:
I'm only referring to how this looks in the docs. In the [file storage docs](https://godoc.org/github.com/deis/logger/storage/file), for example, `NewStorageAdapter` returns `*adapter`, which is meaningless from the perspective of the docs (one of the go linter tools warns about this). And even if the package is only meant to be used by code in this repository, not reusable, even people working on this codebase will still look at the docs 😄